guides/source/api_documentation_guidelines.md

```diff
@@ -60,11 +60,11 @@ end
 
 Communicate to the reader the current way of doing things, both explicitly and implicitly. Use the idioms recommended in edge. Reorder sections to emphasize favored approaches if needed, etc. The documentation should be a model for best practices and canonical, modern Rails usage.
 
-Documentation has to be concise but comprehensive. Explore and document edge cases. What happens if a module is anonymous? What if a collection is empty? What if an argument is nil?
+Documentation has to be concise. Explore and document edge cases. What happens if a module is anonymous? What if a collection is empty? What if an argument is nil?
 
-The proper names of Rails components have a space in between the words, like "Active Support". `ActiveRecord` is a Ruby module, whereas Active Record is an ORM. All Rails documentation should consistently refer to Rails components by their proper name, and if in your next blog post or presentation you remember this tidbit and take it into account that'd be phenomenal.
+The proper names of Rails components have a space in between the words, like "Active Support". `ActiveRecord` is a Ruby module, whereas Active Record is an ORM. All Rails documentation *should* consistently refer to Rails components by their proper names.
```

Comment:
I'm unclear on the purpose of the asterisks...?

When I read that sentence aloud with vocal emphasis on "should", it sounds like I am saying "All Rails documentation *probably* consistently refers to Rails components by their proper names. (It *should*, but I don't know if it actually does.)"

Without the asterisks, it sounds like I am stating a fact, with the expectation that the fact will remain true.  (i.e. I expect the audience to uphold the truth of that statement.)